include/swift/AST/ASTDemangler.h

```diff
@@ -193,6 +193,8 @@ class ASTBuilder {
                                                     unsigned size,
                                                     unsigned alignment);
 
+  Type maybeWrapInExistential(Type type, bool forRequirement);
```

Comment:
I think you can remove the forRequirement parameter entirely, and change the caller to only call maybeWrapInExistential() when forRequirement is false.

Reply:
Sure!